src/Facade/ResourceBuilders/StoresPalletResourceBuilder.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Linn.Stores2.Facade.ResourceBuilders
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using Linn.Common.Facade;
+    using Linn.Common.Resources;
+    using Linn.Stores2.Domain.LinnApps.Stock;
+    using Linn.Stores2.Resources;
+
+    public class StoresPalletResourceBuilder : IBuilder<StoresPallet>
+    {
+        public StoresPalletResource Build(StoresPallet pallet, IEnumerable<string> claims)
+        {
+            var locationId = new StorageLocationResourceBuilder().Build(pallet.LocationId, claims);
+
+            var locationType = pallet.LocationType != null
+                ? new LocationTypeResourceBuilder().Build(pallet.LocationType, claims)
```

Comment:
can probably inject an instance of LocationTypeResourceBuilder here as opposed to constructing one